ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -623,6 +628,49 @@ def osd_encryption_verification():
             )
             raise ValueError("OSD is not encrypted")
 
+    if ocs_version != version.VERSION_4_6:
```

Comment:
you need to change this condition..
for example 4.5!=4.6
```
if ocs_version >= version.VERSION_4_6:

```

Reply:
I want it to skip for version 4.6, thats why I used that condition. since this feature isn't back ported to 4.6 or I can do is 
`if ocs_version > version.VERSION_4_6`